components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorService.java

```diff
@@ -155,8 +152,7 @@ public AccessTokenInfo getTokens(String appName, String grantType,
                 // Access token for authorization code grant type
                 APIMAppConfigurations appConfigs = ServiceReferenceHolder.getInstance()
                         .getAPIMAppConfiguration();
-                String callBackURL = appConfigs.getApimBaseUrl() + AuthenticatorConstants.AUTHORIZATION_CODE_CALLBACK_URL + appName
-                        + "?uiService=" + uiServiceURL;
+                String callBackURL = appConfigs.getApimBaseUrl() + AuthenticatorConstants.AUTHORIZATION_CODE_CALLBACK_URL + appName;
```

Comment:
better to have debug log with the callbackurl

Reply:
Done with https://github.com/wso2/carbon-apimgt/pull/4896/commits/131693463cb521f3aa19ae56aab16f2c65550c2c